app/controllers/members_controller.rb

```diff
@@ -24,6 +24,20 @@ def create
     end
   end
 
+  def confirm_email
+    @member = Member.find_by_confirm_token(params[:id])
+    if @member
+      @member.email_activate
+      MemberMailer.welcome_email(@member).deliver
+      flash[:success] = "Welcome to the Sample App! Your email has been confirmed.
+      Please sign in to continue."
+      redirect_to action: :index
+    else
+      flash[:error] = "Sorry. User does not exist"
```

Comment:
The message here is that the token is invalid, not that user doesn't exist, no? 
Don't feel strongly about it, but maybe fix the message a bit: "Sorry, user doesn't exist.".